### PR TITLE
Automated Tests - CommerceProductCard.testcase - combined PRs 

### PR DESCRIPTION
**Pull request by kristinonias**
https://issues.liferay.com/browse/COMMERCE-5858
https://issues.liferay.com/browse/COMMERCE-5865
https://issues.liferay.com/browse/COMMERCE-5866
https://issues.liferay.com/browse/COMMERCE-5867
https://issues.liferay.com/browse/COMMERCE-5872
https://issues.liferay.com/browse/COMMERCE-5871
https://issues.liferay.com/browse/COMMERCE-5868

These should be ready for review.

cc: @lorenzolapescara

### CODE REVIEW

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
Review started

File: modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path [hunk 1]
```diff
@@ -509,6 +524,16 @@
...
+<tr>
+	<td>PRODUCT_CARD_NAME_AND_PRICE</td>
+	<td>//p[@title='${card_name}']//following-sibling::p[@class='card-text']//span[@class='price-value']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>DEFAULT_PRODUCT_IMAGE</td>
+	<td>//div[contains(@style,'default')]</td>
+	<td></td>
+</tr>
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path:536]: This locator is too generic, please rename it to `PRODUCT_CARD_DEFAULT_IMAGE` and try to identify and use a unique and reusable path(e.g. by using a variable).

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 2]
```diff
@@ -109,12 +140,42 @@ definition {
...
+		CPCommerceCatalog.newProductsEntry(
+			productCatalog = "Minium",
+			productName = "Simple T-Shirt",
+			productType = "Simple");
+
+		Navigator.openWithAppendToBaseURL(urlAppend = "web/minium");
+
+		CommerceNavigator.gotoFrontStoreProductPage(productName = "Simple T-Shirt");
+
+		AssertElementPresent(locator1 = "CommerceEntry#DEFAULT_PRODUCT_IMAGE");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase:178]: As per comment above, please update `CommerceEntry#DEFAULT_PRODUCT_IMAGE` locator name.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 3]
```diff
@@ -278,7 +278,31 @@ definition {
...
+
+		Click(
+			key_navUnderline = "Members",
+			locator1 = "NavUnderline#NAV_UNDERLINE_LINK");
+
+		CommerceAcceleratorsInitializer.inviteNewMemberWithRoleToAccount(
+			emailAddress = "buyer@liferay.com",
+			role = "Buyer");
+
+		ApplicationsMenu.gotoSite(site = "Minium");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase:298]: The test does not include these steps.

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by kristinonias** [comment]
New PR [#1067](https://github.com/liferay-commerce/liferay-portal/pull/1067)

**Pull request closed without merging**